src/dali_executor/executor.test.cc

```diff
@@ -29,15 +29,49 @@
 
 namespace triton { namespace backend { namespace dali { namespace test {
 
+template<typename T, typename Op>
+void pointwise_compare(const std::vector<OBufferDescr> &obuffers,
+                       const std::vector<std::vector<T>> &ibuffers, size_t inp_size, const Op &op) {
+  size_t inp_buff_i = 0;
+  size_t inp_i = 0;
+  size_t out_buff_i = 0;
+  size_t out_i = 0;
+  std::vector<T> obuffer;
+  for (size_t i = 0; i < inp_size; ++i) {
+    if (inp_i == ibuffers[inp_buff_i].size()) {
+      inp_i = 0;
+      inp_buff_i++;
+    }
+    if (out_i == obuffers[out_buff_i].size / sizeof(T)) {
```

Comment:
Is it by any chance possible that a single value would be split?

Reply:
No. The use case for chunked outputs is the the dynamic batching. Each of the outputs buffer will be a response for a single request, so, in fact, not even a sample can be split.